pyvista/core/errors.py

```diff
@@ -0,0 +1,9 @@
+"""Pyvista specific errors."""
+
+
+class NotAllTrianglesError(Exception):
```

Comment:
I'd expect this to be a subtype of `ValueError`:

> exception ValueError
> 
> Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.

Reply:
Good point.